file_name: "grab_leg_6_dmp"
